### PR TITLE
align usages

### PR DESCRIPTION
**Pull request by orrgottlieb**
Align usages in prep for poppoins/figtree

### CODE REVIEW

File: src/components/AttentionBox/AttentionBox.tsx [hunk 1]
```diff
@@ -72,33 +75,33 @@ const AttentionBox: React.FC<AttentionBoxProps> & {
...
-              )}
-              ignoreFocusStyle
-              iconSize="24"
-              iconLabel={iconLabel}
-            />
-          )}
-          <span
+      {/* @ts-ignore */}
+      <Flex
+        /* @ts-ignore */
```
Comment (SergeyRoyt) [inline src/components/AttentionBox/AttentionBox.tsx:80]: Let me merge Flex PR https://github.com/mondaycom/monday-ui-react-core/pull/1006 first, and remove these ts-ignores?

File: src/components/AttentionBox/AttentionBox.tsx [hunk 2]
```diff
@@ -72,33 +75,33 @@ const AttentionBox: React.FC<AttentionBoxProps> & {
...
+          <Icon
+            iconType={iconType}
+            ariaHidden
+            clickable={false}
+            icon={icon}
             className={cx(
-              `${ATTENTION_BOX_CSS_BASE_CLASS}__title-container__title`,
-              `${classNameWithType}__title-container__title`
+              `${ATTENTION_BOX_CSS_BASE_CLASS}__title-container__icon`,
+              `${classNameWithType}__title-container__icon`
```
Comment (SergeyRoyt) [inline src/components/AttentionBox/AttentionBox.tsx:94]: Spot a little misalignment in storybook, I guess it shouldn't look like exactly this
Before:
image

Now:
image

**Review by SergeyRoyt — APPROVED** [review]

File: src/components/AttentionBox/AttentionBox.tsx (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/AttentionBox/AttentionBox.tsx:80]: Merged

File: src/components/AttentionBox/AttentionBox.tsx (hunk 2 above)
Comment (orrgottlieb) [reply src/components/AttentionBox/AttentionBox.tsx:94]: nice catch!! will fix now

Comment (orrgottlieb) [reply src/components/AttentionBox/AttentionBox.tsx:94]: @SergeyRoyt done and merging

**Pull request merged**